xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -847,7 +847,7 @@ void AddCurrentArtTypes(std::vector<std::string>& artTypes, const CVideoInfoTag&
   std::map<std::string, std::string> currentArt;
 
   if (tag.GetAssetInfo().GetId() >= 0)
-    db.GetArtForAsset(tag.m_iFileId, tag.m_iDbId, tag.m_type, currentArt);
+    db.GetArtForAsset(tag.m_iFileId, tag.m_iDbId, tag.m_type, false, currentArt);
```

Comment:
You know I do not like bool parameters in public interfaces. ;-) What does this new `false` tell me? I need to look up the method declaration in the respective header file to get to know. If you would use an enum class with speaking values, understanding this code would be much easier. Any change ti change the bool param to an enum class param?

Reply:
I saw this coming but am still thinking about a better way. Likely will be a different overload of GetArtForAsset with less parameters when fallback is not wanted. Inside videodb it would still be the same.

Reply:
and some doxy...

Reply:
> Likely will be a different overload of GetArtForAsset with less parameters

That would be even more elegant, yes.

Reply:
the overload turned out cumbersome, then I simplified the function, in the end it's an enum.